scripts/debug_tools/prepare_all_cmd.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
```

Comment:
Is it a good idea to hardcore python3? Most of the code should work both with python3 and 2. We should not require both of them to be installed. Once all our dependencies are python3 compatible we can stop supporting python2 and move on to 3 only.

Reply:
Ok, changed it back to default `usr/bin/env python`